app/main/index.js

```diff
@@ -170,11 +174,12 @@ function createMainWindow() {
 		icon: iconPath(),
 		minWidth: 600,
 		minHeight: 400,
+		titleBarStyle: 'hidden-inset',
 		webPreferences: {
 			preload: path.join(__dirname, '../renderer/js/preload.js'),
 			plugins: true,
 			allowDisplayingInsecureContent: true,
-			nodeIntegration: false
+			nodeIntegration: true
```

Comment:
Actually, the only option on the table is to enable nodeIntegration, or we cannot use the WebView element, then the whole idea won't work…

I'm not sure what impact it will have on jQuery… It sounds a bit weird.…